test/test_retriever.py

```diff
@@ -19,6 +20,29 @@
                                        shortname='test')
 test_engine.opts = {'database_name': '{db}_abc'}
 HOMEDIR = os.path.expanduser('~')
+test_dir = os.path.dirname(os.path.abspath(__file__))
+
+
+def create_file(data, output='output_file'):
+    """Takes a string and creates a files for tests"""
+    output_file = os.path.normpath(os.path.join(test_dir, output))
+    with open(output_file, 'w') as testfile:
+        testfile.write(data)
+        testfile.close()
+    return output_file
+
+
+def setup_module():
+    """"change directory to test directory"""
+    dir_path = os.path.dirname(os.path.abspath(__file__))
+    os.chdir(dir_path)
+
+
+def teardown_method():
+    """Cleanup temporary output files after testing"""
+    os.system("rm -r output_*")
+    os.system("rm -r raw_data/MoM2003")
```

Comment:
The `raw_data/MoM2003` directory shouldn't be getting created in this test module should it? If not then we can remove this line.